content/applications/inventory_and_mrp/manufacturing/management/configure_manufacturing_product.rst

```diff
@@ -0,0 +1,73 @@
+===================================
+Configure product for manufacturing
+===================================
+
+.. |BOM| replace:: :abbr:`BoM (Bill of Materials)`
+
+In order to manufacture a product in Odoo *Manufacturing*, the product must be properly configured.
+Doing so consists of enabling the Manufacturing route and configuring a Bill of Materials (BoM) for
+the product. Once these steps are completed, the product is selectable when creating a new
+manufacturing order.
+
+Activate the Manufacture route
+==============================
```

Comment:
```suggestion
Manufacture route activation
============================
```
just another way to word it to keep the proper noun "Manufacture route" capitalized without it seeming un-uniform